src/screens/FundCampaignPledge/FundCampaignPledge.tsx

```
@@ -258,17 +266,15 @@ const fundCampaignPledge = (): JSX.Element => {
                   ) : (
                     <div className={styles.avatarContainer}>
                       <Avatar
-                        key={user._id + '1'}
+                        key={user.id + '1'}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)